### PR TITLE
Add --help to README.md

### PR DESCRIPTION
**Pull request by gitressa**
Adds `--help` to list of options, adds missing colons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,21 +63,22 @@ Arguments
...
   display.
-- `--display=+memory` Set, add or remove elements of the runtime display
+- `--display=+memory`: Set, add or remove elements of the runtime display
   (prefix with `-` or `+` to modify the default set).
-- `--exclude-url=logout` (multiple) Exclude URLs matching the given PCRE pattern.
-- `--header="Foo: Bar"` (multiple) Specify custom header(s).
-- `--include-link=foobar.html` Include given link as if it were linked from the
+- `--exclude-url=logout`: (multiple) Exclude URLs matching the given PCRE pattern.
+- `--header="Foo: Bar"`: (multiple) Specify custom header(s).
+- `--help`: See documentation in this file.
```
Comment (dantleech) [inline README.md:80]: Should be `display application help` I think :)

Comment (gitressa) [reply README.md]: You're right! I updated the patch.

**Comment by dantleech** [comment]
thanks :+1:

**Comment by gitressa** [comment]
Great, thanks!

**Pull request merged**